lib/archethic/account/mem_tables/uco_ledger.ex

```diff
@@ -1,17 +1,27 @@
 defmodule Archethic.Account.MemTables.UCOLedger do
-  @moduledoc false
+  @moduledoc """
```

Comment:
I think the details for the underlying functions should be in their respective doc.
Actually, the function's documentation is checked more often than the module's doc.

Reply:
you are correct. I have removed them